e2e/nomostest/clusters/kind.go

```diff
@@ -202,3 +210,45 @@ apiServer:
 	// We failed to create the cluster maxKindTries times, so fail out.
 	return err
 }
+
+var preloadImages = []string{
+	testing.HTTPDImage,
+	testing.RegistryImage,
+	testing.NginxImage,
+	"prom/prometheus:v2.37.6", // this is defined in a prometheus testdata yaml file
```

Comment:
Seems like kind of a gotcha to hard code this. Can it be parsed out of the testdata file?

Reply:
I wonder if all of them could be parsed out? Or are the others templated?

Reply:
The other server components are defined as Go types rather than yaml manifests, which makes them a bit easier to work with in code - especially for static constants like this. Refactored prometheus installation to use Go types.